code/obj/item/gun/kinetic.dm

```diff
@@ -2144,6 +2144,136 @@ ABSTRACT_TYPE(/obj/item/survival_rifle_barrel)
 		else
 			boutput(user, SPAN_ALERT("You can't fire \the [src] when it is open!"))
 
+#define ONE_BARREL 1
+#define TWO_BARRELS 2
+#define ALL_BARRELS 4
```

Comment:
4 barrels seems insane.

This seems like it turns them into murder hobo loot pinatas.

Reply:
Hopefully 9 safe uses+ longer repair times is better. Let me know if you think we should axe the 4-barrel mode entirely.